generators/entity/templates/src/main/java/package/web/rest/dto/_EntityDTO.java

```diff
@@ -9,7 +9,7 @@
 import java.util.HashSet;
 import java.util.Set;<% } %>
 import java.util.Objects;<% if (databaseType == 'cassandra') { %>
-import java.util.UUID;<% } %><% if (fieldsContainBlob == true) { %>
+import java.util.UUID;<% } %><% if (fieldsContainBlob == true && databaseType != 'mongodb') { %>
```

Comment:
@andidev are you ok with removing Lob for mongo and cassandra? if so ill merge this and amend condition to add only for SQL